pulldown-cmark/specs/math.txt

```diff
@@ -0,0 +1,858 @@
+Run this with `cargo run -- -m -s specs/math.txt`.
+
+## Haskell-CommonMark-Extensions (Pandoc commonmark_x) parity tests
```

Comment:
What does this header refer to? Including the respective spec from commonmark-hs plus maybe some extra parity tests sounds like a good idea, but it shouldn't be at the beginning of the file.

Reply:
I'm also not sure if the first line is correct, since other specs say 'Run this with `cargo test --features gen-tests ...`.'

It would be better having coherence here (I has also noticed that my specs `metadata_blocks.txt` and `heading_attrs.txt` do not have this line, adding it would be a good point).

Reply:
Good spot, that is the outdated command which I corrected at some point in my version of the spec.